internal/olm/operator/registry/index_image.go

```diff
@@ -139,7 +139,7 @@ func (c *IndexImageCatalogCreator) BindFlags(fs *pflag.FlagSet) {
 		"while pulling bundles")
 
 	// default to Restricted
```

Comment:
```suggestion
	// default to Legacy
```

Reply:
Should be resolved as of https://github.com/operator-framework/operator-sdk/pull/6210/commits/1d2e1ed905b49260cb84f3be32cd05d0470d3736